client/app/scripts/superdesk/editor/spellcheck/spellcheck.js

```diff
@@ -123,16 +121,21 @@ function SpellcheckService($q, api, dictionaries, editor) {
     }
 
     /**
-     * Check given text and return all words that are not part of dict
+     * Find errors in given text
+     *
+     * @param {string} text
      */
-    this.check = function check(text) {
+    this.errors = function check(text) {
+
         return getDict().then(function() {
-            var words = text.match(/[a-zA-Z\u00C0-\u1FFF\u2C00-\uD7FF]+/g),
+            var words = text.match(/[0-9a-zA-Z\u00C0-\u1FFF\u2C00-\uD7FF]+/g),
                 errors = [];
             angular.forEach(words, function(word) {
-                var lowerWord = word.toLowerCase();
-                if (!dict[lowerWord]) {
-                    errors.push(word);
+                if (isNaN(word)) {
```

Comment:
Just checking - do we indeed want to interpret strings like `1e3` and `0x5` as numbers and not report them as errors?